src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,13 @@
-export const UserInfo = () => {};
+type Props = {
+  user: User
+};
+
+export const UserInfo:React.FC<Props> = ({ user }) => {
+  const { name, email } = user;
+
+  return (
+    <a className="UserInfo" href={`mailto:${email}`}>
```

Comment:
Consider adding `rel='noopener noreferrer'` to the anchor tag to prevent security vulnerabilities. 🔒